content/en/docs/setup/install/helm/index.md

```diff
@@ -68,10 +63,9 @@ instead with the flags `--set gateways.istio-ingressgateway.type=NodePort`
 appended to the end of the Helm instructions in the installation steps below.
 {{< /tip >}}
 
-### Option 1: Install with Helm via `helm template`
-
-Choose this option if your cluster doesn't have [Tiller](https://helm.sh/docs/topics/architecture/#components)
-deployed and you don't want to install it.
+{{< tip >}}
+The Helm installation method previously detailed here that used the [Tiller](https://helm.sh/docs/topics/architecture/#components) component has been removed. The preferred install path is to use `istioctl` as documented in [Installing with {{< istioctl >}}](/docs/setup/install/istioctl/). If you still desire to use Helm, you'll need to leverage the `helm template` method detailed below.
```

Comment:
If I'm understanding right, it sounds to me like this paragraph is saying that it's best to go use a different process than the one described on this page. True?

If so, then I would recommend pulling that recommendation out of this tip and putting it in a new paragraph right below the "Installation instructions" heading. People often don't read tips, so I suspect that many people who come to this section won't see the line that says to use istioctl.

Reply:
Resolved with https://github.com/istio/istio.io/pull/6847/commits/9381d044356a14493fa33949f50f5c90584581e0